WordPress/theme-experiments
# The Gutenberg Starter Theme as a block-based theme

Here's another one! This uses the [Gutenberg Starter Theme](https://github.com/WordPress/gutenberg-starter-theme) as a base, and fills in the default Gutenberg demo post as homepage starter content. Most of the theme files were stripped out, leaving a the code relatively barebones. 

## Basic steps for testing

1. Create a fresh WordPress site
2. Clone this repository or upload a zip into the site’s themes directory
3. Install the Gutenberg plugin
4. Enable the Full Site Editing experiment in the `Gutenberg > Experiments` panel.
5. Activate the "Gutenberg Starter Theme Blocks" theme from within `Appearance > Themes`.
6. Visit the Customizer and hit "Publish"

Your frontend should render things generally the same way Gutenberg looks with no editor styles.

## Screenshot

![Screen Shot 2020-01-15 at 16 15 27](https://user-images.githubusercontent.com/1202812/72472285-4801b800-37b2-11ea-9d92-ebc7ce7fbd2d.png)
